### ISSUE TITLE
global: better PEP-8 and PEP-257 compliance

### ISSUE THREAD

**Issue by tiborsimko** [inveniosoftware/claimstore#16]
The PEP-8 and PEP-257 compliance should be improved so that we can plug kwalitee checks for Python code in the Travis CI.

```console
claimstore/__init__.py:1:1: D100  Missing docstring in public module
claimstore/wsgi.py:1:1: D100  Missing docstring in public module
claimstore/wsgi.py:2:1: I201 Missing newline before sections or imports.
claimstore/modules/__init__.py:1:1: D100  Missing docstring in public module
claimstore/modules/claims/__init__.py:1:1: D100  Missing docstring in public module
claimstore/modules/claims/views.py:1:1: D100  Missing docstring in public module
claimstore/modules/claims/views.py:1:1: I101 Imported names are in the wrong order. Should be Blueprint, render_template
claimstore/modules/claims/views.py:2:1: I100 Imports statements are in the wrong order. from claimstore.core.json should be before from flask
claimstore/modules/claims/views.py:2:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/views.py:12:1: D103  Missing docstring in public function
claimstore/modules/claims/views.py:17:1: D103  Missing docstring in public function
claimstore/modules/claims/views.py:18:80: E501 line too long (95 > 79 characters)
claimstore/modules/claims/views.py:22:1: D103  Missing docstring in public function
claimstore/modules/claims/views.py:27:1: D103  Missing docstring in public function
claimstore/core/__init__.py:1:1: D100  Missing docstring in public module
claimstore/app.py:1:1: D100  Missing docstring in public module
claimstore/app.py:1:1: I101 Imported names are in the wrong order. Should be Flask, jsonify, render_template
claimstore/app.py:3:1: I100 Imports statements are in the wrong order. from claimstore.core.exception should be before from flask.ext.sqlalchemy
claimstore/app.py:3:1: I201 Missing newline before sections or imports.
claimstore/app.py:10:1: D103  Missing docstring in public function
claimstore/app.py:16:1: D103  Missing docstring in public function
claimstore/core/decorators.py:1:1: D100  Missing docstring in public module
claimstore/core/decorators.py:2:1: I201 Missing newline before sections or imports.
claimstore/core/decorators.py:5:1: D103  Missing docstring in public function
claimstore/core/decorators.py:10:80: E501 line too long (90 > 79 characters)
claimstore/core/exception.py:1:1: D100  Missing docstring in public module
claimstore/core/exception.py:1:1: D101  Missing docstring in public class
claimstore/core/exception.py:4:1: D102  Missing docstring in public method
claimstore/core/exception.py:11:1: D102  Missing docstring in public method
claimstore/core/json.py:1:1: D100  Missing docstring in public module
claimstore/core/json.py:2:1: I100 Imports statements are in the wrong order. import json should be before import os
claimstore/core/json.py:3:1: I201 Missing newline before sections or imports.
claimstore/core/json.py:4:1: I100 Imports statements are in the wrong order. from flask should be before import jsonschema
claimstore/core/json.py:4:1: I201 Missing newline before sections or imports.
claimstore/core/json.py:7:1: D202  No blank lines allowed after function docstring (found 1)
claimstore/core/json.py:10:80: E501 line too long (89 > 79 characters)
claimstore/core/json.py:28:1: D202  No blank lines allowed after function docstring (found 1)
claimstore/modules/claims/models.py:1:1: D100  Missing docstring in public module
claimstore/modules/claims/models.py:2:1: I101 Imported names are in the wrong order. Should be JSONB, UUID
claimstore/modules/claims/models.py:2:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/models.py:3:1: I100 Imports statements are in the wrong order. from claimstore.app should be before from sqlalchemy.dialects.postgresql
claimstore/modules/claims/models.py:3:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/models.py:6:1: D101  Missing docstring in public class
claimstore/modules/claims/models.py:59:1: D102  Missing docstring in public method
claimstore/modules/claims/models.py:63:1: D101  Missing docstring in public class
claimstore/modules/claims/models.py:86:1: D102  Missing docstring in public method
claimstore/modules/claims/models.py:90:1: D101  Missing docstring in public class
claimstore/modules/claims/models.py:119:1: D102  Missing docstring in public method
claimstore/modules/claims/models.py:123:1: D101  Missing docstring in public class
claimstore/modules/claims/models.py:132:1: D102  Missing docstring in public method
claimstore/modules/claims/restful.py:1:1: D100  Missing docstring in public module
claimstore/modules/claims/restful.py:1:1: I101 Imported names are in the wrong order. Should be Blueprint, jsonify, request
claimstore/modules/claims/restful.py:2:1: I100 Imports statements are in the wrong order. from claimstore.modules.claims.models should be before from flask
claimstore/modules/claims/restful.py:2:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/restful.py:2:80: E501 line too long (87 > 79 characters)
claimstore/modules/claims/restful.py:3:1: I100 Imports statements are in the wrong order. from claimstore.app should be before from claimstore.modules.claims.models
claimstore/modules/claims/restful.py:5:1: I100 Imports statements are in the wrong order. from claimstore.core.decorators should be before from claimstore.core.json
claimstore/modules/claims/restful.py:17:1: D103  Missing docstring in public function
claimstore/modules/claims/restful.py:34:80: E501 line too long (98 > 79 characters)
claimstore/modules/claims/restful.py:48:80: E501 line too long (98 > 79 characters)
claimstore/modules/claims/restful.py:53:1: D103  Missing docstring in public function
claimstore/modules/claims/restful.py:63:80: E501 line too long (86 > 79 characters)
claimstore/modules/claims/restful.py:65:80: E501 line too long (92 > 79 characters)
claimstore/modules/claims/restful.py:67:80: E501 line too long (90 > 79 characters)
claimstore/modules/claims/restful.py:69:80: E501 line too long (90 > 79 characters)
claimstore/modules/claims/restful.py:71:80: E501 line too long (89 > 79 characters)
claimstore/modules/claims/restful.py:74:80: E501 line too long (118 > 79 characters)
claimstore/modules/claims/restful.py:76:80: E501 line too long (87 > 79 characters)
claimstore/modules/claims/restful.py:78:80: E501 line too long (92 > 79 characters)
claimstore/modules/claims/restful.py:100:1: D103  Missing docstring in public function
```

**Comment by tiborsimko** [comment]
Actually, there is still some import cosmetics left to do, so I'm reopening this issue:

```
claimstore/wsgi.py:24:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/views.py:23:1: I101 Imported names are in the wrong order. Should be Blueprint, render_template
claimstore/modules/claims/views.py:24:1: I100 Imports statements are in the wrong order. from claimstore.core.json should be before from flask
claimstore/modules/claims/views.py:24:1: I201 Missing newline before sections or imports.
claimstore/app.py:23:1: I101 Imported names are in the wrong order. Should be Flask, jsonify, render_template
claimstore/app.py:25:1: I100 Imports statements are in the wrong order. from claimstore.core.exception should be before from flask.ext.sqlalchemy
claimstore/app.py:25:1: I201 Missing newline before sections or imports.
claimstore/core/decorators.py:24:1: I201 Missing newline before sections or imports.
claimstore/core/json.py:24:1: I100 Imports statements are in the wrong order. import json should be before import os
claimstore/core/json.py:25:1: I201 Missing newline before sections or imports.
claimstore/core/json.py:26:1: I100 Imports statements are in the wrong order. from flask should be before import jsonschema
claimstore/core/json.py:26:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/models.py:24:1: I101 Imported names are in the wrong order. Should be JSONB, UUID
claimstore/modules/claims/models.py:24:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/models.py:25:1: I100 Imports statements are in the wrong order. from claimstore.app should be before from sqlalchemy.dialects.postgresql
claimstore/modules/claims/models.py:25:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/restful.py:23:1: I101 Imported names are in the wrong order. Should be Blueprint, jsonify, request
claimstore/modules/claims/restful.py:24:1: I100 Imports statements are in the wrong order. from claimstore.modules.claims.models should be before from flask
claimstore/modules/claims/restful.py:24:1: I201 Missing newline before sections or imports.
claimstore/modules/claims/restful.py:26:1: I100 Imports statements are in the wrong order. from claimstore.app should be before from claimstore.modules.claims.models
claimstore/modules/claims/restful.py:28:1: I100 Imports statements are in the wrong order. from claimstore.core.decorators should be before from claimstore.core.json
```

(You can install `flake8-import-order` to catch these in your editor, see https://github.com/inveniosoftware/invenio/issues/2482.)

**Comment by tiborsimko** [comment]
Closed by 006208f6616a459c694216a825c21210ee9ddb64.
